mousestyles/data/tests/test_data.py

```diff
@@ -12,3 +12,13 @@
 def test_all_features_loader():
     all_features = data.load_all_features()
     assert_equal(all_features.shape, (21131, 13))
+
+
```

Comment:
Could we add a one line comment to unit test i.e. _#Checking dimensions of test all features load dataset are correct after importing"_